foreman_plugin_template.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.files = Dir['{app,config,db,lib,locale,webpack}/**/*'] + ['LICENSE', 'Rakefile', 'README.md', 'package.json']
   s.test_files = Dir['test/**/*'] + Dir['webpack/**/__tests__/*.js']
 
-  s.required_ruby_version = '>= 2.7'
+  s.required_ruby_version = '>= 2.7', '< 4'
```

Comment:
Looks like we already have dropped support for Ruby versions < 2.7 so the only remaining change here is to set an upper bound to `< 4`. It would be good for the commit message/PR description.